notebooks/Visualize.ipynb

```diff
@@ -219,6 +219,8 @@
     "\n",
     "from rikai.viz import Style\n",
     "\n",
+    "boxes[0] = boxes[0] + \"label\"\n",
```

Comment:
does it change the output?

Reply:
yes, we should see something different after it released.